app/mailers/candidate_mailer.rb

```diff
@@ -141,7 +141,6 @@ def new_offer_made(application_choice)
     @provider_name = @course.provider.name
     @course_name_and_code = @application_choice.current_course_option.course.name_and_code
     @application_form = @application_choice.application_form
-    @show_deadline_reminder = (CycleTimetable.decline_by_default_date - 4.weeks).before? Time.zone.now
```

Comment:
We decided we should show this to candidates as we approach the end of cycle. Has this decision been reverted already?

Reply:
As discussed, it looks like your ticket was created before it was agreed to add the show_deadline_reminder content. Let's keep it in :)